src/ralph/assets/models/components.py

```diff
@@ -1,12 +1,19 @@
 # -*- coding: utf-8 -*-
+import re
+
+from django.core.validators import RegexValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
 
 from ralph.admin.autocomplete import AutocompleteTooltipMixin
 from ralph.assets.models.base import BaseObject
-from ralph.assets.models.choices import ComponentType
+from ralph.assets.models.choices import ComponentType, EthernetSpeed
 from ralph.lib.mixins.fields import NullableCharField
-from ralph.lib.mixins.models import NamedMixin
+from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
+
+MAC_RE = re.compile(r'^\s*([0-9a-fA-F]{2}[:-]){5}[0-9a-fA-F]{2}\s*$')
```

Comment:
in ralph 2 we have mac addresses without `:` - we should discuss how do we want to store them. imo we should store them without `:` and optionally parse mac form field to skip `:`